<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>new_repo_dialog</class>
 <widget class="QDialog" name="new_repo_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>492</width>
    <height>320</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Enter a repo name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="text_repo_name"/>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>enter repo descriptions:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="text_repo_desc"/>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="btn_box">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
